### ISSUE TITLE
Couldn't reconstruct when using trained model.

### ISSUE THREAD

**Issue by rin634** [taldatech/soft-intro-vae-pytorch#13]
Hi Daniel,

While training a model using the script of `train_soft_intro_vae.py`,  it  reconstructed  images  in a good way.

However, when I loaded the trained model in jupyter lab and tried to reconstruct images in order to check, it generated something chaos.

Could you tell me how to reconstruct images using  the trained model 

Thank you

 images (real, recon, fake) while training
![image_130760](https://user-images.githubusercontent.com/52065996/136644498-edf9bbfa-0953-4f38-a3ba-076051b08cd6.jpg)

images (real ,recon) which  the trained model generated
![tes](https://user-images.githubusercontent.com/52065996/136644508-7b56c0bc-289e-4ea5-9970-257ec7be43ed.jpg)

**Comment by taldatech** [comment]
Hi,
This seems a very cool (and diverse) dataset. I assume the first image is what was generated during the training process. In this case the model is in `model.train()` mode which affects BatchNorm. When you load the trained model, try to not use `model.eval()` (see example notebook in the tutorial directory in this repository). Another thing you can try to avoid it is to remove BatchNorm from the architecture (as in the style architecture). See a previous discussion: https://github.com/taldatech/soft-intro-vae-pytorch/issues/8

Another thing I can see from these images is that the is imbalance between `beta_kl` and `beta_rec`, as the reconstructions are very good but the generations seem to be not diverse at all (something like "mode collapse"). In this case I'd recommend to lower `beta_rec` or to increase `beta_kl`. What do the training graphs look like?
